app/main/views/suppliers.py

```diff
@@ -441,7 +441,7 @@ def update_supplier_framework_details(supplier_id, framework_slug):
     if 'agreementDetails' in update_json and framework.framework_agreement_details is None:
         abort(400, "Framework '{}' does not accept 'agreementDetails'".format(framework_slug))
 
-    # Try and find an existing FrameworkAgreement to update, if not then create one based on data from SupplierFramework
+    # Try and find an existing FrameworkAgreement to update, if not then create one
```

Comment:
We probably don't need this comment at all now - seems straightforward enough from the code.  Although if we're going to block creation of a new FrameworkAgreement by this route in future it might be worth mentioning that in a comment.